.github/workflows/build_and_test_wheels.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+on:
+  workflow_call:
+
+jobs:
+  build-and-test-wheels:
+    needs:
+      - wheel-build-amd64
+      - wheel-build-arm64
+      - wheel-test-amd64
+      - wheel-test-arm64
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/pr-builder.yaml@branch-23.12
+  wheel-build-amd64:
+    runs-on: linux-amd64-cpu4
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
```

Comment:
We'll want to build for Python 3.9-3.12 at some point. We may need new `ci-wheel` images for that.

Reply:
right - we did a bunch of playing around with which containers to use early on and ended up rolling with the current matrix mostly _because_ that's what the existing images support on the rapids side. Ultimately however `pynvjitlink` has consumers outside of rapids that do require other versions of python as you mention, so we'll eventually either need to move away from rapids images or create rapids images beyond what rapids supports more broadly just for building this package. Not sure which would be better.

Reply:
I opened https://github.com/rapidsai/miniforge-cuda/pull/55 to help start this process.